### ISSUE TITLE
Better error message when toast does not have permissions to run without sudo

### ISSUE THREAD

**Issue by CodeGradox** [stepchowfun/toast#246]
**Description**
I was going through the guide when I got the error "Unable to pull image" without any details.
Turns out I was just missing sudo...
However, it might be helpful for users who are new to docker, toast or Linux to get a more helpful error message.

Here is the output I got.

```
$ toast
[INFO] Ready to run 1 task: greet.
[INFO] Running task greet…
[ERROR] Unable to pull image. Details:
```
I can run toast without sudo on osx so there is no issue there.

**System**
Arch Linux 5.1.6
Docker 18.09.6-ce
rustc 1.34.2
Toast 0.24.0

**Comment by stepchowfun** [comment]
Thank you for this bug report! I will take a look this weekend.

**Comment by stepchowfun** [comment]
Fixed by https://github.com/stepchowfun/toast/pull/248 and shipped in v0.25.0.
